assets/src/components/v2/pre_fare_single_screen_alert.tsx

```diff
@@ -0,0 +1,288 @@
+      className="alert-card__pio-text__icon"
+      src={imagePath("clock-negative.svg")}
+    />
+    {issue && <div className="alert-card__pio-text__main-text">{issue}</div>}
+    {remedy && <div className={classWithModifier("alert-card__pio-text__secondary-text", pioSecondaryTextSize)}>{remedy}</div>}
+  </>
+)
+
+const standardIssueSection = (issue: string, location: string | null, textSize: string) => (
+  <div className="alert-card__issue">
+    <img
+      className="alert-card__icon"
+      src={imagePath("no-service-black.svg")}
+    />
+    <div>
+      <div className={classWithModifier("alert-card__content-block__text", textSize)}>
+        {issue}
+      </div>
+      {location && <div className="alert-card__issue__location">{location}</div>}
+    </div>
+  </div>
+)
+
+const downstreamIssueSection = (issue: string) => (
+  <div className="alert-card__issue">
+    <div className={classWithModifier("alert-card__content-block__text", "medium")}>{issue}</div>
+  </div>
+)
+
+const remedySection = (effect: string, remedy: string | null, contentTextSize: string) => (
+  <div className="alert-card__remedy">
+    { effect === "shuttle" ?
+      <>
+        <div className="alert-card__remedy__shuttle-icons">
+          <img
+            className="alert-card__icon"
+            src={imagePath("bus-black.svg")}
+          />
+          <img
+            className="alert-card__isa-icon"
+            src={imagePath("ISA_Blue.svg")}
+          /> 
+        </div>
+        <div>
+          <div className={classWithModifier("alert-card__content-block__text", contentTextSize)}>
+            {remedy}
+          </div>
+          <div className="alert-card__body__accessibility-info--text">
+            All shuttle buses are accessible
+          </div>
+        </div>
+      </>
+      : <>
+        <img
+          className="alert-card__icon"
+          src={imagePath("nearby.svg")}
+        />
+        <div className="alert-card__remedy__text">{remedy}</div>
+      </>
+    }
+  </div>
+)
+
+const mapSection = () => {
+  <></>
+}
+
+const PreFareSingleScreenAlert: React.ComponentType<PreFareSingleScreenAlertProps> = (alert) => {
+  const { cause, region, effect, issue, location, remedy, routes, unaffected_routes, updated_at } = alert;
+
+  /** 
+  * This switch statement picks the alert layout
+  * - fallback: icon, followed by a summary & pio text, or just the pio text
+  * - multiline: icon + route pill + text explaining the lines that are closed at the station
+  *              and then icon + route pill + text explaining normal service. Finally, the map section
+  * - standard: icon + issue, and icon + remedy, and then map section
+  * - downstream: map, issue without icon, then icon + remedy
+  **/
+  const layoutRenderer = () => {
+    switch(true) {
+      case effect === "delay":
+        return fallbackLayout()
+      case (effect === "station_closure" || effect === "stop_closure") && region === "here":
+        return multiLineLayout()
```

Comment:
In this case, I am seeing a banner but I don't think I should. Also, `stop_closure` is a bus alert effect. Can it be removed from the conditional?